### ISSUE TITLE
file management

### ISSUE THREAD

**Issue by jimka2001** [scalacenter/scastie#547]
When I press save in the scastie, tool where does it save to?
Can it look at a list of things I've saved?
If I loose the url to a page, can I find it again?
I see there's a snipits list, but I don't see the relation the entries in this list, and the states of my
window after pressing the save button, so I am guessing that Save does not create a snipit.
Can someone help me with the use model here?

<img width="246" alt="Screenshot 2022-01-23 at 00 45 45" src="https://user-images.githubusercontent.com/6414129/150659077-3aed4e3f-a0b8-4756-96a4-2277811c969c.png">

**Comment by vincenzobaz** [comment]
Hello @jimka2001 ! Scastie saves snippets to a MongoDB server managed by the Scala Center.
The code related to storage is in the package `com.olegych.scastie.storage` in the `storage` module. For testing, an in-memory implementation is provided too.

If you log in when using Scastie, it will be able to remember your snippets:
![image](https://user-images.githubusercontent.com/6325220/150694014-48cca829-8d6b-44c7-bfbc-45c33041486d.png)

What do you mean by:

>  I don't see the relation the entries in this list
?

Would you like to contribute to Scastie?

**Comment by jimka2001** [comment]
It could be that this is a bug, because I am indeed logged in.
Yet when I Save, I don't see any updates to the snippits list.
<img width="305" alt="Screenshot 2022-01-23 at 20 44 20" src="https://user-images.githubusercontent.com/6414129/150695353-883d9a85-0912-4b05-9fb7-60afc1d3cd9a.png">

**Comment by jimka2001** [comment]
Experimenting a bit more,  it behaves a bit differently than it did earlier.
If I change the code in the editor, and press save, then activates the snippits menu item,
I don't see the new snippit listed.  However, if I press the reload button on the web browser, I DO SEE
the new snippit listed.

Could this just be a bug in the Safari support?

**Comment by jimka2001** [comment]
Would it be possible to annotate the snippit list a bit?  Either some user text?  or at least the date/time/time-zone the snippit was saved?

**Comment by OlegYch** [comment]
@jimka2001 right, we should reload snippet list after save automatically and add some more info in the list
